torchtext/data/batch.py

```diff
@@ -58,6 +62,20 @@ def __str__(self):
     def __len__(self):
         return self.batch_size
 
+    def _get_vars(self, target):
```

Comment:
Could this be changed to some enumeration and/or commented (ex `target="outputs"`, or just add a docstring)